statsmodels/genmod/families/family.py

```diff
@@ -475,7 +479,7 @@ def resid_anscombe(self, endog, mu, scale=1.):
            \mu_i^{1/6}
         """
         return ((3 / 2.) * (endog**(2 / 3.) - mu**(2 / 3.)) /
-                (mu ** (1 / 6.) * scale ** 0.5))
+                (mu ** (1 / 6.) * (scale / var_weights) ** 0.5))
```

Comment:
It's better to avoid dividing by var_weights
at the cost of adding a separate sqrt we can have var_weight equal to zero for some observations.
(I'm not sure we fully support that, but it's always better to avoid unnecessary division by something that could be zero or close to zero if possible.)